libcaf_io/caf/io/basp/connection_state.test.cpp

```diff
@@ -0,0 +1,61 @@
+// This file is part of CAF, the C++ Actor Framework. See the file LICENSE in
+// the main distribution directory for license terms and copyright or visit
+// https://github.com/actor-framework/actor-framework/blob/master/LICENSE.
+
+#include "caf/io/basp/connection_state.hpp"
+
+#include "caf/test/outline.hpp"
+
+using namespace caf::io::basp;
+
+namespace {
+
+OUTLINE("critical connection states require a connection shutdown") {
```

Comment:
Shouldn't we use the newer `TEST` structure instead?

Reply:
This is from the new testing framework. 🙂

See https://github.com/actor-framework/actor-framework/tree/master/libcaf_test#scenario-outlines.

Reply:
Thanks for the clarification.